docs/UserGuide.md

```diff
@@ -141,80 +141,122 @@ Format: `clear`
 Adds a supplier to the address book.
 
 Format: `add -s n/NAME p/PHONE e/EMAIL com/COMPANY [t/TAG]…​ [pro/PRODUCT]…​`
-<box type="tip" seamless>
 
-Parameters:
+<box type="details" seamless>
 
-- `n/NAME`: Must be alphanumeric, and must not be blank.
-- `p/PHONE`: Must be numeric, must contain at least 3 digits, and must not be blank.
-- `e/EMAIL`: Must be a valid email address, and must not be blank.
-- `com/COMPANY`: Must be alphanumeric, and must not be blank.
-- `[t/TAG]`: Must be alphanumeric, and must be between 1 and 50 (inclusive) characters long.
-- `[pro/PRODUCT]`: Must be alphanumeric, can include spaces, and must be between 1 and 50 (inclusive) characters long.
+Parameters:
 
-**Tip:** A supplier can have any number of tags and products (including 0)
+- `n/NAME`: The supplier's name. It must be alphanumeric, and cannot be blank.
+- `p/PHONE`: The supplier's phone number. It must be numeric, and contain at least 3 digits.
+- `e/EMAIL`: The supplier's email address. It must be in a valid email address format, and cannot be blank.
+  - Please see [below](#valid-email-address-format) for more information on what constitutes a valid email address format.
+- `com/COMPANY`: The company associated with the supplier. It must be in a valid company name format, and cannot be blank.
+  - Please see [below](#valid-company-name-format) for more information on what constitutes a valid company name format.
+- `[t/TAG]`: The tag(s) associated with the supplier. It must be alphanumeric, and only contain between 1 and 50 (inclusive) characters.
+- `[pro/PRODUCT]`: The product(s) associated with the supplier. It must be alphanumeric, only contain between 1 and 50 (inclusive) characters, and spaces are also allowed.
```

Comment:
if we just say spaces are allowed, would this allow tester to just add 5 spaces and claim that it is a valid product but the app does not accept it?

Reply:
We can stat that it must start with a alphanumeric character